README.md

```diff
@@ -35,3 +35,44 @@ Where
 - w: if supplied, will wipe the postgres image
 - m: if supplied, will reset all repos to main
 - path/to/workspace/dir: absolute path to your local directory where all the repos are checked out. Expects them all named the same as the git repos, eg. `funding-service-design-assessment-store`.
+
+# Running in debug mode (VS Code)
+The containers in the docker runner can be run with python in debug mode to allow a debugger to connect. This gives instructions for connecting VS Code.
+
+Each app in docker-compose has the following value for `command`, which makes the debugger run:
+
+        command: ["sh", "-c", "pip install debugpy -t /tmp && python /tmp/debugpy --listen 0.0.0.0:5678 -m flask run --no-debugger --host 0.0.0.0 --port 8080"]
+
+The 'no-debugger' part relates to the flask debugger, it's useful to remove this option if you want to see the stack traces for things like jinja template errors.
```

Comment:
I wonder if we can make this overridable? e.g. via an argument in the docker compose file? So that we still get the Flask debugger by default if you're not using an external one?